src/org/traccar/protocol/IdplProtocolDecoder.java

```diff
@@ -94,14 +94,14 @@ protected Object decode(
 
         position.set(Position.KEY_SATELLITES, parser.nextInt());
         position.set(Position.KEY_RSSI, parser.nextInt());
-        parser.next(); // vehicle status
+        position.set("vehicleStatus", parser.next()); // vehicle status
         position.set(Position.KEY_POWER, parser.nextInt());
         position.set(Position.KEY_BATTERY, parser.nextDouble());
         if (parser.nextInt() == 1) {
             position.set(Position.KEY_ALARM, Position.ALARM_SOS);
         }
-        parser.nextInt(); // body tamper
-        parser.nextInt(); // ac status
+        position.set("bodyTamper", parser.nextInt()); // body tamper
```

Comment:
Reverted code as I am not prepared at this time to determine how to test the string to appropriately raise the alarm.